js/bittrex.js

```diff
@@ -49,10 +49,12 @@ module.exports = class bittrex extends Exchange {
                     'account': 'https://{hostname}/api',
                     'market': 'https://{hostname}/api',
                     'v2': 'https://{hostname}/api/v2.0/pub',
+                    'v3': 'https://api.bittrex.com/v3',
                 },
                 'www': 'https://bittrex.com',
                 'doc': [
                     'https://bittrex.github.io/api/',
+                    'https://bittrex.github.io/api/v3',
```

Comment:
I thought, maybe we better just detach it into a separate file?

Reply:
On a second thought, this may be a way to migrate seamlessly as well, though...

Reply:
I think we're going to need a mix of v2 and v3 in this case anyway

Reply:
... and v1.1!